src/Aspire.Hosting.AWS/Constants.cs

```diff
@@ -27,4 +27,29 @@ internal static class Constants
     internal const string IsAspireHostedEnvVariable = "ASPIRE_HOSTED";
 
     internal const string LambdaPreviewMessage = "Local Lambda development feature is still in active development. Check out the following GitHub issue for status: https://github.com/aws/integrations-on-dotnet-aspire-for-aws/issues/17";
+    
+    /// <summary>
+    /// The launch settings profile name prefix
+    /// </summary>
+    internal const string LaunchSettingsNodePrefix = "Aspire_";
+    
+    /// <summary>
+    /// The launch settings file name
+    /// </summary>
+    internal const string LaunchSettingsFile = "launchSettings.json";
+    
+    /// <summary>
+    /// The target framework of the executable wrapper project
+    /// </summary>
+    internal const string ExecutableWrapperProjectTargetFramework = "net8.0";
```

Comment:
See my comment below that this should not be a constant but we should use the target framework the Lambda project is using.